### DISCUSSION TITLE
Blazor HotKeys not intercepting in WPF CEF App?

### DISCUSSION THREAD

**Discussion by tseanf** [cefsharp/CefSharp#4836] [General]
Hello,

I just started trying to implement CEF in a fresh WPF App.  Nothing fancy, just the control on a window.  The site I am trying to hit uses this library: https://jsakamoto.github.io/Toolbelt.Blazor.HotKeys2/

But when using my page that uses that library, or the library's demo page, keys do not seem to get intercepted.  I am not sure what's up.  This works great in traditional browsers (chrome proper, edge, safari, etc)

I have tried making sure that the browser control is focused (note keyboard presses work in input fields), I have also tried intercepting key up and key down on the window and passing that to browser.GetBrowser().GetHost().SendKeyEvent(keyEvent) - but again, no go.

Any thoughts?  Thanks!

**Comment by amaitland** [comment]
What version are you using?

Test with the `CEF Sample application` see https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

The links are out of date, try getting the latest version https://cef-builds.spotifycdn.com/index.html#windows64

Make sure to run with the `WPF/OffScreen` command line args
